moto/dynamodb/exceptions.py

```diff
@@ -383,6 +383,12 @@ def __init__(self) -> None:
         super().__init__(er, "NUMBER_VALUE cannot be converted to String")
 
 
+class ValidationException(JsonRESTError):
```

Comment:
I saw a similar `ValidationException` in a lot of other modules, but there wasn't one in `dynamodb/` yet.